docs/DeveloperGuide.md

```diff
@@ -508,8 +509,23 @@ testers are expected to do more *exploratory* testing.
  
     2. Test case: `list r/`<br>
        Expected: All archived clients listed. Success message shown. "Archived Client List" is shown below the success message.
-       
+       The color of the priority indicator for each archived client is darkened. 
        ![List All Archived Clients](images/listAllArchivedClients.png)
+       
+### Adding a client
+
+1. Adding a client while the active clients are being shown.
+    
+    1. Prerequisites: List all active clients (see [List Active Clients](#listing-all-active-clients)). Client does not exist. (A client does not exist
+    as long as there is not a client in the clientlist which does not have the same name AND same email or phone number)
```

Comment:
Perhaps this would be clearer:
The client is not already in the client list, i.e. a client that is different from those already in the client list should be added.
2 clients are considered different if 
- They have different names OR
- They have the same name BUT have different email or phone number

Reply:
ok will do :D

Reply:
Approving this first but uniqueness is pending